xml/System.Windows.Forms/HighDpiMode.xml

```diff
@@ -13,7 +13,7 @@
     <BaseTypeName>System.Enum</BaseTypeName>
   </Base>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Specifies the high DPI mode.</summary>
```

Comment:
We should extend what this class is used for. Specifies the high DPI mode of what?

Reply:
Maybe like this?
```suggestion
    <summary>Provides possible high DPI modes for an application.</summary>
```

@dreddy-work @merriemcgaw

Reply:
@RussKie @mairaw the last suggestion might need some rewording. Maybe:

```suggestion
    <summary>Provides all the possible high DPI modes for an application.</summary>
```
Pending approval from @dreddy-work and @merriemcgaw

Reply:
This properties behavior is tied with OS version(IsWindows10_1703OrGreater). Should we specify OS version in the documentation?

Reply:
Also, at any point only one High DPI mode is applied to the application.  So, "Provide high DPI mode for the application" seem more suitable in this case.

Reply:
I'd go with the following then. Let me know what you think.

```suggestion
    <summary>Specifies the different high DPI modes that can be applied to an application.</summary>
```